aten/src/ATen/templates/aten_xla_type.h

```diff
@@ -4,19 +4,5 @@
 #include <ATen/Tensor.h>
 
 namespace ${cpp_namespace} {
-
-// Base ATEN Type class where the XLA specific overrides should be defined.
-class AtenXlaType {
- public:
-  static void InitializeAtenBindings();
```

Comment:
I noticed that this function actually doesn't do anything, so I opted to kill it rather than leave it as a function in the template that all backends need to implement. @ailzhang does that seem fine?

Reply:
Yup this looks like a legacy that we can safely kill :D